lib/design/app_colors.dart

```diff
@@ -1,7 +1,31 @@
 import 'package:flutter/material.dart';
 
 class AppColors {
-  // NEW COLORS, ASK GERY BEFORE ADDING A NEW COLOR-----DarkGreen1
+  /// Primary Colors
```

Comment:
Still messy, but all this will be replaced with the theme colors and neat and clean